pkg/rate/api_limiter_test.go

```diff
@@ -794,3 +794,24 @@ func (b *ControllerSuite) TestReservationCancel(c *check.C) {
 		req2.Done()
 	}
 }
+
+func (b *ControllerSuite) TestSetRateLimit(c *check.C) {
```

Comment:
New tests should use the testing framework instead of the checkmate, unless strictly necessary.